tests/pfc_asym/test_pfc_asym.py

```diff
@@ -5,6 +5,10 @@
     pytest.mark.topology('t0')
 ]
 
+@pytest.fixture(scope='module', autouse=True)
+def prepare_syncdrpc(swapSyncd):
+    pass
```

Comment:
@antonptashnik Why define a pass function here?

Reply:
here `prepare_syncdrpc` is defined to make `swapSyncd` to be auto-executed once before all suite tests, so in fact it does not need body. It's only use is to set scope and autouse for `swapSyncd` to be called for PFC asym test module